sbnana/SBNAna/Cuts/NumuCuts.cxx

```diff
@@ -31,11 +31,11 @@ namespace ana{
   
   const Cut kNumuTrkLen([](const caf::SRSliceProxy* slc)
 			{ // TO DO: Prim pandora trk len primary_track.length > fConfig.TrackLength
-			  bool hastrk = ( slc->reco.ntrk > 0 );
+			  bool hastrk = ( slc->reco.npfp > 0 );
 			  if (!hastrk) return hastrk;
 			  
 			  unsigned int muIdx = (unsigned int)kPrimMuonIdx(slc);
-			  double len = slc->reco.trk[muIdx].len;
+			  double len = slc->reco.pfp[muIdx].trk.len;
```

Comment:
I guess that `muIdx` can now feasibly be `-1` so should be checked

Reply:
I can put in a check for muIdx > 0 like exists in other parts of this repo, which should fix any issues

Reply:
Any updates on implementing this?